ui/src/adapters/api/credentials.ts

```diff
@@ -86,17 +91,19 @@ export async function getCredential({
 
 export async function getCredentials({
   env,
-  params: { did, query, status },
+  params: { did, max_results, page, query, status },
   signal,
 }: {
   env: Env;
   params: {
     did?: string;
+    max_results?: number;
+    page?: number;
```

Comment:
Why `maxResults` and `page` are optional?

Reply:
Why should they be required? The API applies some default values.